cluster-operator/src/main/resources/cluster-roles/020-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -86,7 +86,6 @@ rules:
       - patch
       - update
   - apiGroups:
-      - "" # legacy core events api, used by topic operator
       - "events.k8s.io" # new events api, used by cluster operator
     resources:
       # The cluster operator needs to be able to create events and delegate permissions to do so
```

Comment:
Does it still delegate the events? Or are they now used only in the CO?

Reply:
Both TO and UO do not create any event, so delegation is not required. Do you want me to update the comment?

Reply:
Yeah, I guess that would be perfect. Thanks.